colcon_ansi_colors/output_style/catkin_tools.py

```diff
@@ -75,7 +75,7 @@ def apply_style(self, style):  # noqa: D102
         style.PackageOrJobName = AnsiEscape.Cyan
         style.Pictogram = AnsiEscape.Bright + AnsiEscape.Green
         style.SectionStart = AnsiEscape.Default
-        style.SectionEnd = AnsiEscape.Bright + AnsiEscape.Black
+        style.SectionEnd = AnsiEscape.Faint + AnsiEscape.Default
```

Comment:
Much better and more readable on a dark terminal:
<img width="400"  alt="Screenshot from 2025-08-07 20-08-20" src="https://github.com/user-attachments/assets/8a63b14c-0b8b-477f-9d40-632c1b08a411" />